presidio-analyzer/presidio-venv/bin/python

```diff
@@ -0,0 +1 @@
+python3.7
```

Comment:
This file should not be committed

Reply:
The entire presidio-venv should be deleted